src/app/screens/Login/layout.js

```diff
@@ -1,30 +1,28 @@
 import React from 'react';
 import { ImageBackground, Text, TextInput, TouchableOpacity } from 'react-native';
 import { lightGray } from '../../../constants/colors';
-
 import styles from './styles';
 import backgroundImage from '../../assets/bc_start.png'
 
 function Login() {
+    const commonProps = { style: styles.input, numberOfLines: 1, placeholderTextColor: lightGray };
```

Comment:
you could extract this outside the function so you don't create this constant each time the function is called